__tests__/commands/_helpers.js

```diff
@@ -131,3 +131,11 @@ export async function run<T, R>(
     throw new Error(`${err && err.stack} \nConsole output:\n ${out}`);
   }
 }
+export const setPlatform = (platform : string) : string => {
+  const previous = process.platform;
+  Object.defineProperty(process, 'platform', {
```

Comment:
I wish Node.js had a decent dependency injection framework, this is super hacky 😛